src/index.html

```diff
@@ -1,16 +1,59 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+
+  <body class="page__body">
+
+    <div class="container">
+      <!-- #region formbig-->
+      <form
+        action="post"
+        class="form"
+        data-qa="big"
+        >
+        <button
+          id="icon-search-big"
+          class="icon-search icon-search--size-big form__search-button-big"
```

Comment:
Wrong BEM naming:

If the Block is "form" and the Element is "search-button", why is it also called "icon-search" ?
Correct naming would look like: "form__search-button search-button search-button--size-big".

Reply:
I'll make the chagens, in fact I use the worong DOM element to represent the button in project. Thank you for adivice.